auction.sympl

```diff
@@ -33,95 +40,324 @@ schema Auction:
     creator: str
     closed: bool
 
-# Returns all the auctions in our storage
+# Admins are the only users allowed to:
+# 1. Create the Premium auction channel
```

Comment:
I'm not sure I fully understand the premise of a private auction. I wonder if private bids would be more appropriate, i.e. a silent auction? Not sure how easy that would be to implement.

Reply:
So all of the auctions are public, but bids are posted to a channel that only the auction creator can read.